docs/src/icons/SvgCoursera.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+function SvgComponent(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
Are you sure about using React components for them? Why not use SVGs icons that can be lazy-loaded?

Reply:
it is removed.